docs/references/sql-introspection.md

```diff
@@ -0,0 +1,129 @@
+---
+sidebar_position: 3
+description: "API reference for inspecting the invocation status and service state."
+---
+# SQL Introspection API
+
+This page contains the reference of the introspection tables.
+To learn how to access the instrospection interface, check out the [instrospection documentation](/operate/introspection).
+
+## Table: `state`
+
+| Column name | Type | Description |
+|-------------|------|-------------|
+| `partition_key` | `UInt64` | Internal column that is used for partitioning the services invocations. Can be ignored. |
+| `service_name` | `Utf8` | The name of the invoked service. |
+| `service_key` | `Utf8` | The key of the Virtual Object. |
+| `key` | `Utf8` | The `utf8` state key. |
+| `value_utf8` | `Utf8` | Only contains meaningful values when a service stores state as `utf8`. This is the case for TypeScript services since the TypeScript SDK serializes values as JSON. |
```

Comment:
`This is the case for TypeScript services since the TypeScript SDK serializes values as JSON. ` --> I think Java as well now, with the JsonSerdes?

Reply:
Will add this.